source/diagrams/understand-diagrams.txt

```diff
@@ -109,6 +109,43 @@ features of relationships:
 
 - A prong (or "crow's feet") indicates "many" in the relationship.
 
+Undo and Redo Functionality
+---------------------------
+
+Relational Migrator diagrams support undo and redo functions. Using 
+these options with a selected diagram reverses or reapplies the 
+previous diagram action. You can use either the diagram toolbar or 
+keyboard shortcuts to perform these actions.
+
+Toolbar
+~~~~~~~
+
+You can use the :icon-lg:`Undo` and :icon-lg:`Redo` buttons on the 
+diagram toolbar:
+
+.. image:: /img/diagrams/undo-redo.png
```

Comment:
(suggestion) 
I understand why you added this and it does make things really clear. However, I think the icons are enough. What do you think about omitting the screenshot in this case?

Reply:
I think adding the diagram centers the user and reinforces where they should be looking. The UI is quite a busy one, and its possible that additional undo redo buttons may be added (in mapping rules or in a sub modal) so in that regard having the screen shot future proofs the edit and it does not take much screen real estate.

Reply:
Ok. Sounds fine and thanks for thinking it through.